ipwb/util.py

```diff
@@ -146,10 +146,30 @@ def datetimeToRFC1123(digits14):
         newLocale = 'en_US.utf8'
 
     locale.setlocale(locale.LC_TIME, newLocale)
+
+
+def datetimeToRFC1123(digits14):
```

Comment:
Name of this method should reflect that it is the opposite of `rfc1123ToDigits14()`

Reply:
I agree. Stand by.

Reply:
Updated in latest. Investigating a TravisCI tests issue that may have the unit test names hard-coded. Please continue the review.